pingcap/tidb-operator
# charts: add local backend support for TiDB-Lightning

### What problem does this PR solve?

add `local` backend support for TiDB-Lightning Helm chart, may close #3162.

### What is changed and how does it work?

- create a PVC for `local` backend
- update some config items

### Code changes


### Tests


  1. set `backend: local`
  2. `helm install` the chart
  3. verify the data has been imported into the TiDB cluster

### Side effects


### Related changes


### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
Add local backend support for the TiDB-Lightning chart
```
